Vagrantfile

```diff
@@ -2,8 +2,8 @@ Vagrant.require_version ">= 2.2.2"
 
 Vagrant.configure(2) do |config|
   config.vm.box = "ubuntu/bionic64"
-  config.vm.network "private_network", ip: "172.30.1.5"
-  config.ssh.insert_key = false
+  config.vm.network "public_network", ip: "192.168.0.27", bridge: "Broadcom 802.11n Network Adapter"
+  config.ssh.insert_key = true
```

Comment:
Can you revert your changes to this file please, if these changes were for a good reason please can you submit a separate PR?

Reply:
reverted